tests/sentry/api/endpoints/test_user_notification_fine_tuning.py

```diff
@@ -240,82 +170,54 @@ def test_saves_and_returns_deploy(self):
             == "0"
         )
 
-        update = {}
-        update[self.org.id] = 1
-        resp = self.client.put(url, data=update)
+        self.get_valid_response("me", "deploy", method="put", status_code=204, **{self.org.id: 2})
```

Comment:
it looks like the org ID is 1 here, not 2

Reply:
Good catch 👍🏽